cd/utils/mxnet_base_image.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# -*- coding: utf-8 -*-
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+mxnet_variant=${1:?"Please specify the mxnet variant as the first parameter"}
+
+case ${mxnet_variant} in 
+    cu80*)
+    echo "nvidia/cuda:8.0-cudnn7-runtime-ubuntu16.04"
+    ;;
+    cu90*)
+    echo "nvidia/cuda:9.0-cudnn7-runtime-ubuntu16.04"
+    ;;
+    cu91*)
```

Comment:
I always pick on this one... but isn't it not supported?

Reply:
Good catch! I'll remove